css/decanter-grid.css

```diff
@@ -265,6 +266,4 @@ body {
   margin-left: 85.2980433528%; }
 
 .decanter-offset-eleven-twelfths {
-  margin-left: 93.8278476881%; }
-
-/*# sourceMappingURL=decanter-grid.css.map */
```

Comment:
@josephgknox 
It looks like the source map gets removed with the auto-prefixer. Is this going to cause an error for debugging?